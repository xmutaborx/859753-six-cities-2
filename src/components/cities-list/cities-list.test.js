import React from 'react';
import renderer from 'react-test-renderer';
import CitiesList from './cities-list';

it(`Cities List renders correctly`, () => {
  const tree = renderer
    .create(<CitiesList
      cities={[`first city`, `second city`]}
      changeCity={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
