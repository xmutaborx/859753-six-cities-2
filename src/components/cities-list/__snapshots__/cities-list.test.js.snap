// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cities List renders correctly 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    >
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item"
          href="#"
          onClick={[Function]}
        >
          <span>
            first city
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item"
          href="#"
          onClick={[Function]}
        >
          <span>
            second city
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
